nautobot/core/tests/test_events.py

```diff
@@ -83,37 +82,42 @@ def test_publish_events_to_broker(self):
         deregister_event_broker(event_broker)
         deregister_event_broker(event_broker_2)
 
+    @override_settings(
+        NAUTOBOT_EVENT_BROKERS={
+            "SyslogEventBroker": {
+                "CLASS": "nautobot.core.events.SyslogEventBroker",
+                "TOPICS": {
+                    "INCLUDE": ["nautobot.test.*"],
+                },
+            }
+        }
+    )
```

Comment:
Modifying the settings here does not seem to work

Reply:
Given the current pattern, might need to add some internal/test APIs like `unregister_all_event_brokers` or `load_event_brokers(reload=True)`?